src/fileTransformer/fileTransformer.ts

```diff
@@ -9,6 +10,17 @@ export type Tools = {
    * This does a simple string.replace but will throw an error if the given pattern does not match anything.
    */
   replaceContent(params: { file: File; pattern: RegExp; replacement: string }): string
+  /**
+   * Tools designed specifically for working with Prisma Schema.
+   */
+  prismaSchema: {
+    /**
+     * Add a preview flag to the Prisma Client generator block.
+     *
+     * Upsert semantics are used: If preview flags are already present then this one is appended. If there are no preview flags yet then the preview flags field is added.
+     */
+    addPreviewFlag(params: { file: File; previewFlag: PreviewFlag }): string
```

Comment:
This "tool" abstracts away a fair bit of incidental code to get the job done. File transformers should stay more or less declarative.